theano/sandbox/cuda/dnn.py

```diff
@@ -2005,6 +2005,11 @@ def dnn_pool(img, ws, stride=(1, 1), mode='max', pad=(0, 0)):
 
     """
     img = gpu_contiguous(img)
+    if mode == "sum":
+        ret = GpuDnnPool(mode="average_inc_pad")(img, ws, stride, pad)
+        window_elem = theano.tensor.prod(ws).astype(ret.dtype)
+        return as_cuda_ndarray_variable(ret * window_elem)
+
```

Comment:
Currently it crash. I didn't wanted to test a crash fix when I can just
copy/paste the code from the new back-end. There was new ops.

On Thu, Sep 8, 2016 at 12:13 PM, abergeron <notifications@github.com> wrote:

> In theano/sandbox/cuda/dnn.py
> <https://github.com/Theano/Theano/pull/4932#discussion_r78038114>:
>
> > @@ -2005,6 +2005,11 @@ def dnn_pool(img, ws, stride=(1, 1), mode='max', pad=(0, 0)):
> >
> >      """
> >      img = gpu_contiguous(img)
> > +    if mode == "sum":
> > +        ret = GpuDnnPool(mode="average_inc_pad")(img, ws, stride, pad)
> > +        window_elem = theano.tensor.prod(ws).astype(ret.dtype)
> > +        return as_cuda_ndarray_variable(ret * window_elem)
> > +
>
> Didn't we decide to not add new features in the old backend?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4932/files/4aab58b485d96dfee74701110abf6d971b9dcdaf#r78038114>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AALC-zVqgsMIpkJmUwzetu6XDU5Wclanks5qoDQdgaJpZM4Jz8eJ>
> .
>